src/frontEnd/vueChecks/createSummary.js

```diff
@@ -1,5 +1,205 @@
+    value: 'N/A'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-6',
+    description:
+      'El proyecto usa webpack para generar el build en producción y babel para los imports con alias',
+    value: reports.find(({ metric }) => metric === vueMetrics.USE_CLI_SERVICE).value
+  });
+};
+
+const uiUxSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-UI-UX-1',
+    description: 'El proyecto es mobile friendly según lighthouse',
+    value: reports.some(
+      elem => elem.metric === seoMetrics.LIGHTHOUSE_PWA_OVERALL && elem.value >= limits.pwaMin
+    )
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-2',
+    description: 'El proyecto usa Sass respetando el linter correspondiente',
+    value: reports.some(elem => elem.metric === eslintMetrics.ESLINT_CONFIG && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-3',
+    description: 'El proyecto posee internacionalización',
+    value: reports.some(elem => elem.metric === generalMetrics.I18N && elem.value >= limits.i18nPercentage)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-4',
+    description: 'Proyecto respeta la estructura de directorios sugerida',
+    value: reports.filter(elem => elem.metric === generalMetrics.FOLDER_STRUCTURE).every(elem => elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-5',
+    description: 'El proyecto usa vue-loader para generar componentes de un solo archivo SFC',
+    value: reports.some(elem => elem.metric === vueMetrics.USE_VUEX && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-7',
+    description: 'SFC que no estén al nivel de app deben estar scoped',
+    value: reports.some(elem => elem.metric === vueMetrics.SCOPED_STYLES && elem.value)
+  });
+};
+
+const clientServerSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-CLIENT-SERVER-1',
+    description: 'El proyecto posee services dedicados a los distintos recurso que posee',
+    value: reports.some(
+      elem =>
+        elem.metric === generalMetrics.FOLDER_STRUCTURE && elem.description.includes('services') && elem.value
+    )
+  });
+
+  summary.push({
+    metric: 'SUMMARY-CLIENT-SERVER-2',
+    description: 'Se utiliza una configuración de apisauce / axios para cada API que se comunique',
+    value: reports.some(elem => elem.metric === dependenciesMetrics.AXIOS_APISAUCE && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-CLIENT-SERVER-4',
+    description: 'Usa Vuex store para manejar el estado de la aplicación ',
+    value: (() => {
+      let hasVuexDependency = false;
+      let hasVuexImport = false;
+
+      reports.foEarch(elem => {
+        if (elem.metric === dependenciesMetrics.VUEX && elem.value) {
+          hasVuexDependency = true;
+        }
+        if (elem.metric === vueMetrics.USE_VUEX) {
+          hasVuexImport = true;
+        }
+      });
+
+      return hasVuexDependency && hasVuexImport;
```

Comment:
You can do this:
```
reports.some(elem.metric === dependenciesMetrics.VUEX && elem.value) && 
reports.some(elem.metric === dependenciesMetrics. USE_VUEX && elem.value)
```

Reply:
Your suggestion is so clear and valid, but not performant, because you're doing 2 loops for just one validation that it's possible to do with just one. Do you still wanna change that?

Reply:
Okey noted, in that case i think that what you want to use is a `reduce`

Reply:
I'm gonna use:

```
summary.push({
  metric: 'SUMMARY-CLIENT-SERVER-4',
  description: 'Usa Vuex store para manejar el estado de la aplicación ',
  value: reports.some(({ metric, value }) => metric === vueMetrics.USE_VUEX && value)
});
```
Because if the project use Vuex inherently it have it as depency.